[CS2113-W15-4] DinerDirector

kristianachwan opened the pull request:
DinerDirector (DD) is a terminal app that helps restaurant managers to manage their restaurants in a more convenient manner, optimized for use via **Command Line Interface (CLI)**

firwer commented on docs/uml/images/ParserSequenceDiagram.png:
Would it be better if you include this part of the sequence into your sequence diagram? (loop)
```
while (!isExit) {
            String userInput = ui.getUserInput();
            Command command = new Parser().parseCommand(userInput);
            assert command instanceof Command : "command should be a command class";
            command.execute(ui);
            isExit = command.isExit();
        }
```

firwer commented on docs/uml/images/ParserSequenceDiagram.png:
Maybe you can also include a simple overview for the Ui component into your sequence diagram as well if it doesn't complicate your diagram too much

firwer commented on docs/uml/AddMeetingCommandSequenceDiagram-Add_Meeting_Sequence_Diagram.png:
Just curious, I read the code for AddMeetingCommand but didn't see anything that suggest multi-threading, why is there a parallel in the sequence diagram?

firwer commented on docs/uml/AddDishCommandSequenceDiagram.puml:
```diff
@@ -0,0 +1,22 @@
+@startuml
+title Add Dish Sequence Diagram
+
+'actor User
```
Just a small remark, for this one, is there a reason why User is commented out? Or maybe you can consider using another participant that is calling this AddDishCommand so that reader knows where is this call initiated from

firwer commented on docs/uml/images/AddDishCommandSequenceDiagram-Add_Dish_Sequence_Diagram.png:
The highlighted part should be deactivated i think?
![image](https://user-images.githubusercontent.com/7589432/228429254-d215ce2c-819c-494e-9f61-e1087476c051.png)

firwer reviewed (commented):
Generally okay, i understand it is still work in progress, the explanations are rather clear and good, some small remarks related to diagrams. Maybe include a little more diagrams would be goodSSS

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+An `isExit` method will return a boolean value that decides whether the program should exit after this command is called.
+
+
+### Manager Component (PeiHao)
```
I would recommend removing your names from the headings! Makes it look more cohesive

Toh-HongFeng replied:
Adding a diagram would make it easier to visualize how the component works.

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+   If the addition was a success, a message will be printed through TextUi class to notify the reader that the deadline object has been added.
+
+
+####deleteDeadline
```
header is broken

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+## Design 
+### Architecture
+![Architecture_Diagram](uml/images/DinerDirectorArchitectureDiagram.png)
```
Imo this image is a bit too big, you could reduce its size. Currently, it spans twice the height of my computer screen

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+The `Staff` Feature allows user to create, read, update, delete (CRUD) `Staff` objects in the list of staffs. 
+`Staff` list is managed by `StaffManager` Class similar to other entity objects. Here are hot it works for each functionality: 
+
+1. add_staff  
```
Your type of bullet point (ie. numbered vs the black dots) is inconsistent throughout the entire DG. I recommend using a particular style for each type of content, ie. describing the functions vs the way a command is executed, or use the same type of bullet point throughout.

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ...                                                 | I want to ...                                            | So that I can ...                                                                         |
```
I really liked the user stories and I think they all fit the purpose of your product. However, it would be better to make it more consistent. For the `so that i can...` section, some of the individual cells start with `so that i can...` or `i can...`. I would recommend removing these to make it more consistent throughout and to be in-line with table headings

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
 # Developer Guide
```
It would be good to include a table of contents! I found it hard to navigate through the document and to figure out what particular section I'm looking at.

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+
+### Command Component (Zheng Rong)
+
+##### The command component consists of the following:
```
does this need to be a heading? Seems a little unnecessary to me, especially since it is not used in the rest of the document

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+### Manager Component (PeiHao)
+The manager component consists of four different managers, 
+in which the list of entity is initialized and the methods implementing the entity are written inside.
+- DeadlineManager: This class contains an ArrayList of deadlines and methods implementing the deadlines like addDeadline, printDeadline and deleteDeadline.
```
I recommend putting the various components or commands into `this code format` as it makes it a bit more readable

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+2. With the `userInputSplit[]`, the `0` index will be extracted out. That will be used as identification for the command the user typed in.
+3. The `commandWord` will be used in the switch statement to select the appropriate command. returning `IncorrectCommand` class is the default behavior.
+4. If the `commandWord` is valid, it will run the appropriate `prepareXYZCommand()`.
+5. Each of the individual `prepareXYZCommand()` will take in the userInput without the command portion. The variable is named `userInputNoCommand`. The `prepareXYZCommand()` will check the userInput to see if all the appropriate values are added, and return `XYZCommand` class if the values are correct. `prepareXXXCommand()` will return `IncorrectCommand` class if there are some missing values or  ...
```
Where did the `prepareXXXCommand()` come from? I think it should be `prepareXYZCommand()`

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+An ArrayList of meetings is initialized in the MeetingManager. Three methods that implement the meetings are also inside.
+- addMeeting: Add a meeting to the meeting list.
+  ![](./uml/AddMeetingCommandSequenceDiagram-Add_Meeting_Sequence_Diagram.png)
+- deleteMeeting: Delete a meeting in the meeting list by its index.
```
I think you're missing some diagrams here

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+   A deadline object can only be removed if the index is a valid index in the list.
+   If the deletion was a success, a message will be printed through TextUi class to notify the reader that the deadline object has been deleted.
+
+3. printDeadlines
```
I think it would be good if you're more consistent in your way of listing methods.

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+
+### Utils Component (Darren)
+
+![UtilsParserClassDiagram](uml/images/UtilsParserClassDiagram.png)    
```
The (C) should be removed from your diagrams, and you should use `+`/`-`/`#`.

This goes for your other class diagrams too.

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+## Implementation
+### Parsing Feature (Darren)
+
+![ParserSquenceDiagram](./uml/images/ParserSequenceDiagram.png)  
```
Remember to close your activation bars!

natashatanyt commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+The 'Meeting' class in the entity package shows the attributes that a meeting object has: time and issue, both are of String type.
+An ArrayList of meetings is initialized in the MeetingManager. Three methods that implement the meetings are also inside.
+- addMeeting: Add a meeting to the meeting list.
+  ![](./uml/AddMeetingCommandSequenceDiagram-Add_Meeting_Sequence_Diagram.png)
```
As with the remaining sequence diagrams, you're missing a few things:
1. You need a colon at the start of all your instances, ie `:AddMeetingCommand`
2. You need to close your activation bars at the bottom of the diagrams.
3. After a return statement, you should close the activation bar, same w/ your `Dish Feature` component
![image](https://user-images.githubusercontent.com/88123171/228705519-14379a41-c8b1-49d1-bdf3-409b70e7ffa5.png)

Toh-HongFeng replied:
<img width="358" alt="image" src="https://user-images.githubusercontent.com/88129105/228765262-a2fb4860-d542-4576-a818-344811f6c8f9.png">
Are these frames for parallel paths? If so, are they supposed to be connected as a single frame separated by a dotted line?

natashatanyt reviewed (commented):
Overall, good job! Apart from some minor mistakes and not following convention, it seemed quite ok to me.

One thing I would recommend is ensuring that your DG is consistently written to make it seem like it was cohesive and written by a single person.

skyanzy commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+   If the deletion was a success, a message will be printed through TextUi class to notify the reader that the deadline object has been deleted.
+
+3. printDeadlines
+   ![](./uml/ViewDeadlineDiagram.png)
```
<img width="628" alt="image" src="https://user-images.githubusercontent.com/88618401/228719119-9c0ec6a3-afd4-4f2b-b9a0-7133cb09b183.png">
I think there might be some missing activation bars

skyanzy commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+
+The Dish feature consists of three functions:
+
+##### Adding dish to list:
```
I think the headers used are inconsistent.
e.g. the subheaders under your deadline feature used #### while here ##### is used

bentohset commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+* Parse the command given the user input and extracts out the necessary information related to the command.
+* Returns the appropriate Command Class based on the parsed input.
+
+![UtilsStorageClassDiagram](uml/images/UtilsStorageClassDiagram.png)  
```
I like how you used XYZ to omit complicated details

bentohset commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+
+### Staff Feature 
+The `Staff` Feature allows user to create, read, update, delete (CRUD) `Staff` objects in the list of staffs. 
+`Staff` list is managed by `StaffManager` Class similar to other entity objects. Here are hot it works for each functionality: 
```
Seems to be a typo on the word 'hot'. Should it be 'how' instead?

Toh-HongFeng commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+
+##### The command component consists of the following:
+
+![Command_Package_Diagram](uml/images/CommandPackageDiagram.png)
```
The diagram can also include other information such as the abstract nature of the Command class

Toh-HongFeng commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+1. The `Parser()` class will be called to create a new instance of `Parser`.
+2. Afterwards, when the `parseCommand()` method is called from `DinerDirector` class, the `parseCommand()` will split the given userInput first.
+2. With the `userInputSplit[]`, the `0` index will be extracted out. That will be used as identification for the command the user typed in.
+3. The `commandWord` will be used in the switch statement to select the appropriate command. returning `IncorrectCommand` class is the default behavior.
```
there seems to be a link to the Command class missing from the sequence diagram.

Toh-HongFeng commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,204 @@
+
+##### Adding dish to list:
+
+![](uml/images/AddDishCommandSequenceDiagram-Add_Dish_Sequence_Diagram.png)
```
<img width="290" alt="image" src="https://user-images.githubusercontent.com/88129105/228766230-dfd8b858-32e0-4e56-9e25-a89c84d5b3ab.png">
Do you mean to return before calling execute() ?

darrenangwx committed:
Update Developer Guide

remove unuse user story

darrenangwx committed:
Update User Guide formatting

darrenangwx committed:
Update Developer Guide formatting

kristianachwan committed:
Merge pull request #93 from darrenangwx/branch-Documentation

Update User Guide and Developer Guide formatting

douph810975 committed:
fix meeting bugs

darrenangwx committed:
Add darrenangwx.md for PPP

Stella1585 committed:
fix view deadline command error in UG

Stella1585 committed:
delete irrelevant note in view_deadlines

darrenangwx committed:
Add Style for UML

Stella1585 committed:
no error msg when deadline list empty

darrenangwx committed:
Update contributions to DG

Stella1585 committed:
edit find_deadline error in help message

Stella1585 committed:
typo in UG find_command

darrenangwx committed:
Merge pull request #155 from darrenangwx/branch-Documentation

Add darrenangwx.md for PPP

darrenangwx committed:
Update AboutUs with profile pictures

darrenangwx committed:
Update broken links in AboutUs

darrenangwx committed:
Fix formatting of PPP

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

darrenangwx committed:
Merge pull request #156 from darrenangwx/branch-Documentation

Update AboutUs page

kristianachwan committed:
Update architecture diagram

kristianachwan committed:
Add PPP template

kristianachwan committed:
Limit phone number length

kristianachwan committed:
Make the code more defensive with more strict format

kristianachwan committed:
Add constraint unique name

kristianachwan committed:
Update help

kristianachwan committed:
Fix date formatting

kristianachwan committed:
Add PPP and fix indentation

maanyos commented on src/main/java/dinerdirector/DinerDirector.java:
```diff
@@ -0,0 +1,101 @@
+            ui.printMessage(String.format(Messages.ERROR_CREATE_DIRECTORY, e));
+        }
+
+        try {
+            deadlineStorage.readAndLoadFromDeadlineFile();
+        } catch (FileNotFoundException e) {
+            ui.printMessage(String.format(Messages.ERROR_STORAGE_FILE_NOT_FOUND, DeadlineStorage.FILENAME_DEADLINE));
+        }
+
+        try {
+            meetingStorage.readAndLoadFromMeetingFile();
+        } catch (FileNotFoundException e) {
+            ui.printMessage(String.format(Messages.ERROR_STORAGE_FILE_NOT_FOUND, MeetingStorage.FILENAME_MEETING));
+        }
+
+        try {
+            dishStorage.readAndLoadFromDishFile();
+        } catch (FileNotFoundException e) {
+            ui.printMessage(String.format(Messages.ERROR_STORAGE_FILE_NOT_FOUND, DishStorage.FILENAME_DISH));
+        }
+
+        try {
+            staffStorage.readAndLoadFromStaffFile();
+        } catch (FileNotFoundException e) {
+            ui.printMessage(String.format(Messages.ERROR_STORAGE_FILE_NOT_FOUND, StaffStorage.FILENAME_STAFF));
+        }
```
@darrenangwx 
consider combining into a single try clause?

maanyos commented on src/main/java/manager/MeetingManager.java:
```diff
@@ -0,0 +1,70 @@
+
+    }
+    public static void findMeeting(String keyword, TextUi ui){
+        ArrayList<Meeting> meetingFound=new ArrayList<>();
```
@douph810975 
consider renaming this array list?

maanyos commented on src/main/java/manager/StaffManager.java:
```diff
@@ -0,0 +1,91 @@
+     * @param ui Ui object in if there is anything to be printed.
+     */
+    public static void findStaff(String name,TextUi ui) {
+        ArrayList<Staff> staffFound = new ArrayList<>();
```
@kristianachwan 
consider renaming this array list?

maanyos commented on src/main/java/manager/DishManager.java:
```diff
@@ -0,0 +1,88 @@
+            }
+        }
+
+        String dishesWithStringToFindInList = "";
```
@Zeno-Zr 
This attribute's name is quite long, same with some of the other long names in this class

maanyos commented on src/main/java/manager/DeadlineManager.java:
```diff
@@ -0,0 +1,118 @@
+     * @param ui       manages user output.
+     */
+    public static void addDeadline(Deadline deadline, TextUi ui) {
+        int len = deadlines.size();
```
@Stella1585 
use more meaningful name for variables

maanyos commented on src/main/java/manager/DeadlineManager.java:
```diff
@@ -0,0 +1,118 @@
+     */
+    public static void findDeadline(String keyword, TextUi ui) {
+        ArrayList<Deadline> matchingDeadlines = new ArrayList<>();
+        for (Deadline x : deadlines) {
```
@Stella1585 
default is use the same character as the object name, for example Deadline d

maanyos reviewed (commented):
for all in the group
try to increase abstraction in parser, and move them into the respective constructors, especially in the prepare add command methods
Parser is currently 400 lines and is doing too much
code looks mostly ok in other classes, and quite consistent across different coders also

douph810975 committed:
add PPP and update some test cases

Zeno-Zr committed:
Update limzhengrong.md for PPP

Zeno-Zr committed:
Update printed message for AddDishCommand

* AddDishCommand now prints out the latest dish added when called.

Zeno-Zr committed:
Revert dish_list to 9 items

Zeno-Zr committed:
Update mentions of recipe to Dish in UG

Zeno-Zr committed:
Update findDishCommand message

* findDishCommand now returns a message informing the user that there
are no dishes with matching keyword instead of an empty space.

Zeno-Zr committed:
Update findDishCommand to search for substring

* UG has been updated to reflect the change.

Zeno-Zr committed:
Add exception catching for add dish price

Zeno-Zr committed:
Update indentation in DishManager

Zeno-Zr committed:
Update UG to reflect max price able to key in by user

Zeno-Zr committed:
Update DishStorage with new parser method

* DishStorage now stores input the same way as how a user would enter
* DishStorage now loads dishes with empty ingredient lists properly

Zeno-Zr committed:
Add message that prints out the list is empty when view dish is called

Zeno-Zr committed:
Update indentation and imports

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

darrenangwx committed:
Fix bug command handler

Now able to handle whitespaces infront of commands

darrenangwx committed:
Add excess param check for help and exit command

darrenangwx committed:
Update error message

darrenangwx committed:
Add documentation on how to edit storage files

darrenangwx committed:
Add documentation on how to edit staff_list.txt

Zeno-Zr committed:
Revamp reading and loading of DishStorage

* Instead of doing multistep parsing, a modified regex is used, similar
to how parsing is done in the Parser class for add dish.
* Note: Further testing might be needed, but as of now it handles all
the input in the data file, and does not read in dishes with negative
prices.

Zeno-Zr committed:
Remove unused imports

darrenangwx committed:
Update Style.puml to remove skins

Stella1585 committed:
make sure name before time

Stella1585 committed:
made find command case sensitive

Zeno-Zr committed:
Create UML sequence diagrams for Dish commands

* Add UML sequence diagram .puml files for add dish and view dish
* Refactor methods in DishManager
* Update UG to reflect the correct output when example input is entered
* Update Style.puml to remove unnecessary cosmetic changes

darrenangwx committed:
Update Developer Guide

Removed Skins from UML Diagrams
Add documentation for XYZStorage
Add documentation for Non-Functional Requirements
Update the Appendix A product scope

Zeno-Zr committed:
Add the rest of the Dish UML seqeunce diagrams

* Add Find and delete dishes UML seqeunce diagrams
* Update Add and View dishes UML seqeunce diagrams due to typos

darrenangwx committed:
Fix bug in storage message

darrenangwx committed:
Update User Guide

Add more information to the storage guide

darrenangwx committed:
Fix IO redicrection test

Stella1585 committed:
added ppp, edit dg

Stella1585 committed:
edit msg

douph810975 committed:
shift the print message

Zeno-Zr committed:
Shift message printing add dish and delete dish

darrenangwx committed:
Merge pull request #153 from douph810975/fix_meeting_bugs

fix meeting bugs and add PPP

Stella1585 committed:
edit UMI

Stella1585 committed:
delete comment

darrenangwx committed:
Merge branch 'master' into LimZhengRong-BugFixes

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

darrenangwx committed:
Merge pull request #159 from Zeno-Zr/LimZhengRong-BugFixes

Bug fixes for Dish

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

Stella1585 committed:
add .getmessage for errors

Stella1585 committed:
Update DeadlineManager.java

darrenangwx committed:
Merge branch 'master' into Stella1595-FixBug

darrenangwx committed:
Merge pull request #161 from Stella1585/Stella1595-FixBug

Stella1595 fix bug

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

douph810975 committed:
Merge pull request #160 from darrenangwx/branch-FixBugs

Bug fixes for help, exit, command and command handler

kristianachwan committed:
Move printing message to manager

kristianachwan committed:
Move printing message to manager

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

Zeno-Zr committed:
Merge pull request #162 from darrenangwx/branch-Documentation

Update Developer Guide and User Guide

kristianachwan committed:
Add regex checking to storage

Zeno-Zr committed:
Update message printing in Dish commands

* Remove redundant code left from shifting messages to DishManager.
* Print out the deleted dish instead of just "deleted dish".
* Update message used to print out when list of dishes is empty to
match the rest of the features.

Zeno-Zr committed:
Add checks to prevent numbers ingredients

* Add regex checks to ensure numbers are not added as ingredients in
both Parser and DishStorage.

Zeno-Zr committed:
Update find dish to be case insensitive

kristianachwan committed:
Update architecture diagram

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

Zeno-Zr committed:
Prevent Add dish from adding duplicate dish names

* Update Add dish and DishStorage from adding dishes with duplicate
names.
* Add new method in DishManager to check if the pending dish has a
description matching any dishes in the list of dishes.
* Updates messages to be printed to the console when the user tries to
add a dish with a duplicate name, and/or the dish_list text file
contains dishes with duplicate names.

darrenangwx committed:
Add method headers

Zeno-Zr committed:
Remove unused imports

darrenangwx committed:
Merge pull request #165 from Zeno-Zr/LimZhengRong-BugFixes

Bug fixes for Dish (2nd round)

darrenangwx committed:
Add data folder to .gitignore

darrenangwx committed:
Add data folder to .gitignore

Zeno-Zr committed:
Merge branch 'master' into LimZhengRong-Documentation

Zeno-Zr committed:
Refactor addDishCommand and deleteDishCommand

* Drop the "Command" for the above two methods which were called in
AddDishCommand and DeleteDishCommand

darrenangwx committed:
Merge pull request #158 from Zeno-Zr/LimZhengRong-Documentation

Update limzhengrong.md for PPP

Zeno-Zr committed:
Rename PPP to github username

darrenangwx committed:
Merge branch 'master' into Stella1585-EditDG

darrenangwx committed:
Merge pull request #163 from Stella1585/Stella1585-EditDG

added ppp, edit dg

darrenangwx committed:
Update AboutUs.md

darrenangwx committed:
Merge pull request #167 from Zeno-Zr/LimZhengRong-Documentation

Rename PPP to github username

Zeno-Zr committed:
Merge pull request #164 from darrenangwx/branch-MethodHeaders

Add method headers

Zeno-Zr committed:
rename PPP to lowercase

Zeno-Zr committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp into LimZhengRong-Documentation

Zeno-Zr committed:
Merge pull request #168 from Zeno-Zr/LimZhengRong-Documentation

Rename PPP to lowercase

kristianachwan committed:
Merge branch 'master' into fix_bug_and_update_docu

kristianachwan committed:
Remove unused imports

kristianachwan committed:
Merge pull request #157 from kristianachwan/fix_bug_and_update_docu

Update architecture diagram and fix bug

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

darrenangwx committed:
Merge branch 'master' of https://github.com/darrenangwx/tp into branch-Documentation

darrenangwx committed:
Update help message to follow the User Guide

darrenangwx committed:
Update command table

darrenangwx committed:
Update Sequence Diagram to show the closing of activation bars

darrenangwx committed:
Update size of diagrams

darrenangwx committed:
Update UML diagram

darrenangwx committed:
Update UML diagram

darrenangwx committed:
Update UML diagram size

darrenangwx committed:
Remove diagrams

darrenangwx committed:
Add UML diagrams

kristianachwan committed:
Standardize methods name

kristianachwan committed:
Fix javadoc

kristianachwan committed:
Fix javadoc2

douph810975 committed:
Add manager diagram, add header comments, fix bugs

Stella1585 committed:
edit DG

Stella1585 committed:
Merge branch 'master' into Stella1585-EditDG

Zeno-Zr committed:
Add UML diagrams for Commands

* Add UML diagrams for the commands packages, specifically splitting
them up into two diagrams
* Update the command section of the DG

darrenangwx committed:
Add setup guide in Developer Guide

darrenangwx committed:
Fix formatting

darrenangwx committed:
Add Manual Testing for meeting commands

Zeno-Zr committed:
Add UML dish sequence diagrams to DG

* Update the UML diagram for Add dish.
* Add UML diagrams for View, delete and find dish.
* Update descriptions describing the 4 dish commands.

darrenangwx committed:
Add manual testing for deadline command

darrenangwx committed:
Add manual testing for dish and staff

Zeno-Zr committed:
Update Dish commands and Dish storage in the UG

* Update the format used for DishStorage in the UG.
* Update the descriptions for Dish commands in the UG.

darrenangwx committed:
Add content page to Developer Guide

Zeno-Zr committed:
Add a Purpose section in Dish feature in the UserGuide

Zeno-Zr committed:
Reword and fix formatting of DG

darrenangwx committed:
Fix IO Redirection test

kristianachwan committed:
update DG

Stella1585 committed:
Update Messages.java

darrenangwx committed:
Update default readme

darrenangwx committed:
Update User Guide to add purpose to help command

darrenangwx committed:
Update PPP

darrenangwx committed:
no message

kristianachwan committed:
More descriptive

kristianachwan committed:
Merge pull request #169 from darrenangwx/branch-Documentation

Update User Guide and Developer Guide

Stella1585 committed:
edi header

darrenangwx committed:
Update DeveloperGuide.md

darrenangwx committed:
Update UserGuide.md

darrenangwx committed:
Update MeetingManager.java

kristianachwan committed:
Add extra space for method header

darrenangwx committed:
Update DeveloperGuide.md

darrenangwx committed:
Merge pull request #172 from Stella1585/Stella1585-EditDG

Stella1585 edit dg

douph810975 committed:
update diagrams

Zeno-Zr committed:
Add method headers for Dish related commands

* Add method headers to AddDishCommand, DeleteDishCommand,
FindDishCommand, ViewDishCommand, DishManager, Parser classes.

darrenangwx committed:
Merge pull request #170 from kristianachwan/standardize_methods

Standardize methods name

Stella1585 committed:
add get msg

darrenangwx committed:
Merge pull request #174 from Stella1585/master

add get msg

douph810975 committed:
update diagram

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp into branch-Documentation

Zeno-Zr committed:
Update UML sequence diagrams for Dish commands

darrenangwx committed:
Update Deadline sequence diagram

Zeno-Zr committed:
Add updated Dish UML sequence diagrams to DG

* Add updated UML sequence diagrams to Add, view, delete and find Dish.
* Add colour to Command components.

douph810975 committed:
change variable name

kristianachwan committed:
Fix variable naming

douph810975 committed:
update meeting diagram

darrenangwx committed:
Update Deadline Diagram

Zeno-Zr committed:
Update Dish Sequence diagrams in DG

darrenangwx committed:
Merge branch 'master' into fix_more_bugs_and_update

darrenangwx committed:
Merge pull request #171 from douph810975/fix_more_bugs_and_update

Add manager diagram, add header comments, fix bugs

kristianachwan committed:
Add seq diagram and fix typo

Zeno-Zr committed:
Update Dish UML sequence diagrams with colors

* Update AddDishCommandSequenceDiagram, DeleteDishCommandSequenceDiagram
, FindDishCommandSequenceDiagram, ViewDishCommandSequenceDiagram with
colors in the DG
* Remove unused messages related to Dish in Messages class.

Zeno-Zr committed:
Change formatting of Dish commands headers in DG

darrenangwx committed:
Merge pull request #176 from kristianachwan/Fix_issues_and_seq_diagram

Fix issues and seq diagram

darrenangwx committed:
Merge pull request #173 from Zeno-Zr/LimZhengRong-Documentation

Update DG and UG documentations

darrenangwx committed:
Merge pull request #175 from darrenangwx/branch-Documentation

Update UML diagram

Zeno-Zr committed:
Update individual PPP

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

darrenangwx committed:
Merge branch 'master' of https://github.com/darrenangwx/tp into branch-Documentation

darrenangwx committed:
Merge pull request #177 from Zeno-Zr/LimZhengRong-Documentation

Update individual PPP

darrenangwx committed:
Update UserGuide with staff purpose and formatting

darrenangwx committed:
Fix formatting of Developer Guide

darrenangwx committed:
Fix Developer Guide formatting

darrenangwx committed:
Update user stories to fit the context

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

darrenangwx committed:
Remove data folder

Zeno-Zr committed:
Update bullet points to steps in Dish section of DG

kristianachwan committed:
Off PPP header

Zeno-Zr committed:
Merge branch 'master' into LimZhengRong-Documentation

douph810975 committed:
update DG

douph810975 committed:
update DG

kristianachwan committed:
Update DG

darrenangwx committed:
Update DeveloperGuide.md

Zeno-Zr committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp into LimZhengRong-Documentation

Zeno-Zr committed:
Merge branch 'LimZhengRong-Documentation' of https://github.com/Zeno-Zr/tp into LimZhengRong-Documentation

darrenangwx committed:
Merge pull request #180 from kristianachwan/fix_dg

Fix dg

darrenangwx committed:
Merge pull request #179 from Zeno-Zr/LimZhengRong-Documentation

Update bullet points to steps in Dish section of DG

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp into branch-Documentation

darrenangwx committed:
Fix Developer Guide formatting

Stella1585 committed:
change into step 1,2,3

darrenangwx committed:
Update DeveloperGuide.md

darrenangwx committed:
Merge pull request #182 from Stella1585/master

change into step 1,2,3

darrenangwx committed:
Merge pull request #181 from douph810975/fix_more_bugs_and_update

update DG

kristianachwan committed:
Fix regex bug

kristianachwan committed:
Add empty message

Zeno-Zr committed:
Merge pull request #178 from darrenangwx/branch-Documentation

Update Developer Guide and Remove data folder

kristianachwan committed:
Empty staff_list

kristianachwan committed:
Change dg header

kristianachwan committed:
Delete data

darrenangwx committed:
Merge pull request #183 from kristianachwan/fix_staff_bug

Fix staff bug

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

kristianachwan committed:
Remove logging and change message

kristianachwan committed:
Add intentation

kristianachwan committed:
Update DG

darrenangwx committed:
Update UserGuide.md

edit description for add meeting and deadline

darrenangwx committed:
Update UserGuide.md

darrenangwx committed:
Merge pull request #184 from kristianachwan/fix_staff_parsing

Remove logging and change message

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

kristianachwan committed:
Change view message

kristianachwan committed:
Make staff lowercase

kristianachwan committed:
Catch date parse exception

kristianachwan committed:
Update ug

douph810975 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

# Conflicts:
#	docs/DeveloperGuide.md

douph810975 committed:
no message

douph810975 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

# Conflicts:
#	data/meeting_list.txt

kristianachwan committed:
remove unused imports

darrenangwx committed:
Merge pull request #185 from kristianachwan/fix_staff_parsing

Fix staff parsing

douph810975 committed:
add toLowerCase

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

darrenangwx committed:
Merge pull request #186 from douph810975/optimize

Optimize

kristianachwan committed:
Change UG

darrenangwx committed:
Merge pull request #187 from kristianachwan/fix_ug_staff

Change UG

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

darrenangwx committed:
Update User Guide to fit with current iteration of DinerDirector

kristianachwan committed:
Merge pull request #188 from darrenangwx/master

Update User Guide

darrenangwx committed:
Update Meeting sequence diagram

darrenangwx committed:
Update meeting sequence diagram

darrenangwx committed:
Update staff sequence diagram

kristianachwan committed:
Add end of the line constraint

kristianachwan committed:
Fix tabbing issue

darrenangwx committed:
Update diagrams

darrenangwx committed:
Add UML diagrams

kristianachwan committed:
Update staff storage regex

kristianachwan committed:
Remove unused imports

darrenangwx committed:
Fix formatting and remove unecessary colons

darrenangwx committed:
Remove uncessary colons

darrenangwx committed:
Update content page

darrenangwx committed:
Merge pull request #189 from kristianachwan/Fix_regex_2

Add end of the line constraint

kristianachwan committed:
Merge pull request #190 from darrenangwx/master

Update Developer Guide UML Diagram

darrenangwx committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W15-4/tp

darrenangwx committed:
Update User Guide

darrenangwx committed:
Merge pull request #191 from darrenangwx/master

Update User Guide

Zeno-Zr committed:
Shorten ingredient names to fit PDF for DG

darrenangwx committed:
Merge pull request #192 from Zeno-Zr/LimZhengRong-Documentation

Shorten ingredient names to fit PDF for DG
